std_msgs/Header header

# Number of faces detected
int32 num_face

# Mouth open or closed
bool mouth_open

# Mouth Center
float64 mouth_x
float64 mouth_y

# Mouth Ellipse Axes
float64 a
float64 b

# Mouth Ellipse angle
float64 theta
